# Conveyor belt state message
float64 velocity            # velocity of the belt (m/s, in +Y direction of belt frame)
